CMakeLists.txt

```diff
@@ -276,6 +276,12 @@ endif()
 
 option(WITH_BLUEFS "libbluefs library" OFF)
 
+option(WITH_QAT "Enable Qat driver" OFF)
+if (WITH_QAT)
+  find_package(QatDrv REQUIRED)
+  set(HAVE_QATDRV $(QatDrv_FOUND))
+endif()
+
 option(WITH_QATZIP "Enable QATZIP" OFF)
 if(WITH_QATZIP)
   find_package(qatzip REQUIRED)
```

Comment:
whether will it be better to merge these two options? this will simplify options about QAT, like:
```
option(WITH_QAT "Enable QAT support" OFF)
if(WITH_QAT)
  find_package(QatDrv REQUIRED)
  set(HAVE_QATDRV $(QatDrv_FOUND))
  find_package(qatzip REQUIRED)
  set(HAVE_QATZIP ${QATZIP_FOUND})
endif(WITH_QATZIP)
```

Reply:
Thanks for your comment. However, I am afraid that it won't work for QAT based Encryption which doesn't need QATzip library.  In this case, if a user only need QAT based encryption in Ceph and doesn't install QATzip, Findqatzip.cmake will be still read and processed by CMake _find_package()_. The compilation will stop by CMake because the relevant dependencies cannot be found.